### PR TITLE
arrays/tests

### PR DESCRIPTION
**Pull request by Edgaraszs**
Given task

### CODE REVIEW

File: src/main/java/com/epam/training/array/ArrayLogic.java [hunk 1]
```diff
@@ -1,16 +1,84 @@
...
     }
-    
-   //TODO: add other methods here
 
+    //TODO: add other methods here
+    public void arrayMaxAndMinValues(Array array){
+        System.out.println("Maximum array value: " + maxArrayValue(array) + " " + "Minimum array value: " + minArrayValue(array));
+    }
+
+    public int maxArrayValue(Array array) {
```
Comment (ingenmech) [inline src/main/java/com/epam/training/array/ArrayLogic.java:63]: maxArrayValue wrong naming, you should name method with verb for example findMaxValue()

File: src/main/java/com/epam/training/array/ArrayLogic.java [hunk 2]
```diff
@@ -1,16 +1,84 @@
...
+        int maxValue = maxArray[0];
+        for (int j : maxArray) {
+            if (j > maxValue) {
+                maxValue = j;
+            }
+        }
+        return maxValue;
+    }
+
+    public int minArrayValue(Array array) {
```
Comment (ingenmech) [inline src/main/java/com/epam/training/array/ArrayLogic.java:74]: you should name method with verb for example findMinValue()

File: src/main/java/com/epam/training/array/ArrayLogic.java [hunk 3]
```diff
@@ -1,16 +1,84 @@
...
+
+        for (int prime : primes) {
+            boolean isPrime = true;
+            if (prime == 1) {
+                isPrime = false;
+            }
+            for (int j = 2; j < prime; j++) {
+                if (prime % j == 0) {
+                    isPrime = false;
+                    break;
```
Comment (ingenmech) [inline src/main/java/com/epam/training/array/ArrayLogic.java:42]: don't use "break" in loop "for" read Clean Code.

File: src/main/java/com/epam/training/array/ArrayLogic.java [hunk 4]
```diff
@@ -1,16 +1,84 @@
...
+            resultOfPrimes[i] = primeNumbers.get(i);
+        }
+        return new Array (resultOfPrimes);
     }
-    
-   //TODO: add other methods here
 
+    //TODO: add other methods here
+    public void arrayMaxAndMinValues(Array array){
+        System.out.println("Maximum array value: " + maxArrayValue(array) + " " + "Minimum array value: " + minArrayValue(array));
```
Comment (ingenmech) [inline src/main/java/com/epam/training/array/ArrayLogic.java:60]: you should name method with verb

File: src/test/java/com/epam/training/array/ArrayLogicTest.java [hunk 5]
```diff
@@ -1,7 +1,57 @@
...
+import static org.junit.Assert.*;
+
 public class ArrayLogicTest {
 
+    private final ArrayLogic arrayLogic = new ArrayLogic();
+
     //TODO: add tests for various scenarios
+    @Test
+    public void testIfValueExistsInArray(){
+        //given
```
Comment (ingenmech) [inline src/test/java/com/epam/training/array/ArrayLogicTest.java:15]: Could you rename it for example testMethodShouldDoSomthingWhenDataIsValid()

File: src/test/java/com/epam/training/array/ArrayLogicTest.java [hunk 6]
```diff
@@ -1,7 +1,57 @@
...
+        int[] exists = {2,1,3,5,6};
+        Array array = new Array(exists);
+        //then
+        assertTrue(arrayLogic.exists(array, 5)); // checks if given element exists in given Array
+        assertFalse(arrayLogic.exists(array, 10)); // checks if given element do not exist in given Array
```
Comment (ingenmech) [inline src/test/java/com/epam/training/array/ArrayLogicTest.java:20]: in one test you should test just one statement

File: src/test/java/com/epam/training/array/ArrayLogicTest.java [hunk 7]
```diff
@@ -1,7 +1,57 @@
...
+    }
+
+    @Test
+    public void testForPrimeNumbers(){
+        //given
+        int[] testArray = {1,2,3,4,5};
```
Comment (ingenmech) [inline src/test/java/com/epam/training/array/ArrayLogicTest.java:26]: you may create static variable, because you use this array many times

File: src/test/java/com/epam/training/array/ArraySorterTest.java [hunk 8]
```diff
@@ -1,13 +1,35 @@
...
+        //when
+        arraySorter.sort(array);
+        //then
+        assertArrayEquals(expected, array.getItems());
     }
 
     //TODO: add tests for other positive and negative scenarios
-
+    @Test
+    public void testSortForAllArrayNumbers(){
```
Comment (ingenmech) [inline src/test/java/com/epam/training/array/ArraySorterTest.java:25]: testSortShouldSortArrayWhenData.....

File: src/test/java/com/epam/training/array/ArrayLogicTest.java [hunk 9]
```diff
@@ -1,7 +1,57 @@
...
+        int[] expected = {2,3,5};
+        Array array = new Array(testArray);
+        //when
+        Array result = arrayLogic.findPrimeNumbers(array);
+        //then
+        assertArrayEquals(expected, result.getItems());
+    }
+
+    @Test
+    public void testArrayMaximumValue(){
```
Comment (ingenmech) [inline src/test/java/com/epam/training/array/ArrayLogicTest.java:36]: test....should....when....

File: src/test/java/com/epam/training/array/ArrayLogicTest.java [hunk 10]
```diff
@@ -1,7 +1,57 @@
...
+        int[] testArray = {2,1,3,5,6};
+        Array array = new Array(testArray);
+        //when
+        int max = arrayLogic.maxArrayValue(array);
+        //then
+        Assert.assertEquals(6, max); // checks if 6 is maximum value in array
+    }
+
+    @Test
+    public void testArrayMinimumValue(){
```
Comment (ingenmech) [inline src/test/java/com/epam/training/array/ArrayLogicTest.java:47]: test....should....when....
